css/bs-constants.import.less

```diff
@@ -1,9 +1,9 @@
 /* Some constants copied from Bootstrap's */
 
-@bsInfoColor: #d9edf7;
-@bsSuccessColor: #dff0d8;
-@bsDangerColor: #f2dede;
-@bsBorderColor: #ddd;
+@bsInfoColor: @state-info-text;
+@bsSuccessColor: @state-success-text;
+@bsDangerColor: @state-danger-text;
+@bsBorderColor: @nav-tabs-border-color;
```

Comment:
I'm aware that we were only playing with bsWarningColor, but did we want to leave that out of here? Also, since we're now able to take full advantage of Bootstrap and LESS, should we we even have bsInfoColor/bsSuccessfulColor, etc? We're just referencing another variable in this case, and the bs-constants file was only created so we *could* have some variables. This kind of feels nitpicky.